src/main/java/org/pac4j/dropwizard/J2EHelper.java

```diff
@@ -32,7 +28,15 @@ private J2EHelper() {
     public static void registerSecurityFilter(Environment environment,
             Config config, ServletSecurityFilterConfiguration fConf) {
 
-        final SecurityFilter filter = new SecurityFilter();
+        final SecurityFilter filter = new SecurityFilter(config);
```

Comment:
It seems quite normal to pass in the `config` here.

Reply:
Waiting for @victornoel 's approval...

Reply:
We have to be careful, because the config is set later by a method called `setConfigOnly`, let me check that out :)

Reply:
Yep, we shouldn't be passing the config here because it will make it be set as the singleton config for the whole classpath (see https://github.com/pac4j/j2e-pac4j/blob/master/src/main/java/org/pac4j/j2e/filter/AbstractConfigFilter.java#L99) which is not desirable in the context of dropwizard.